chessboard/script.js

```diff
@@ -0,0 +1,97 @@
+const board = document.querySelector(".board");
+const chess = document.querySelector(".chess");
+const bottom = document.createElement("div");
+const left = document.createElement("div");
+bottom.className = "bottom";
+left.className = "left";
+for (let i = 0; i < 8; i++) {
+  console.log(i, String.fromCharCode(i + 97));
+  const div = document.createElement("div");
+  const div1 = document.createElement("div");
+  const p = document.createElement("p");
+  const p1 = document.createElement("p");
+  p.textContent = String.fromCharCode(i + 97);
+  div.appendChild(p);
+  bottom.appendChild(div);
+  p1.textContent = 8 - i;
+  div1.appendChild(p1);
+  left.appendChild(div1);
+}
+chess.appendChild(bottom);
+chess.appendChild(left);
+console.log(board);
+for (let i = 0; i < 8; i++) {
+  const block = document.createElement("div");
+  block.className = "block";
+  for (let j = 0; j < 8; j++) {
+    const box = document.createElement("div");
+    box.className = "box";
+    box.id = `box` + (i + 1) + "" + (j + 1);
+    block.appendChild(box);
+  }
+  board.appendChild(block);
+}
```

Comment:
The two for loops you wrote, to create the labels of the chessboard, and to create UI of the chessboard can be better written as

```
const createLabelsOfBoard = () => {
  const bottom = document.createElement("div");
  const left = document.createElement("div");
  bottom.className="bottom"
  left.className="left"
  for (let i = 0; i < 8; i++) {
      console.log(i, String.fromCharCode(i+97))
    const div = document.createElement("div");
    const div1 = document.createElement("div");
    const p = document.createElement("p");
    const p1 = document.createElement("p");
    p.textContent=String.fromCharCode(i+97)
    div.appendChild(p)
      bottom.appendChild(div)
      p1.textContent=8-i;
      div1.appendChild(p1)
      left.appendChild(div1)
  }
  chess.appendChild(bottom)
  chess.appendChild(left)  
}

const createBoardLayout = () => {
  for (let i = 0; i < 8; i++) {
  const block = document.createElement("div");
  block.className = "block";
  for (let j = 0; j < 8; j++) {
    const box = document.createElement("div");
    box.className = "box";
    box.id = `box` + (i + 1) + "" + (j + 1);
    block.appendChild(box);
  }
  board.appendChild(block);
}
}

createLabelsOfBoard()
createBoardLayout()
```

I have kept your logic intact, just created helper functions for each and invoked them while page loads, the naming convention also makes the purpose of the function clear